modules/ROOT/pages/update-acm.adoc

```diff
@@ -191,7 +191,7 @@ Configure sharing settings for members:
 . In *Setup*, search for *Users* using the *Quick Find* box and click *Profiles*.
 . Click the *[Community Name] Member User* profile.
```

Comment:
At this point the user can have more than one profile. we should include something saying "if you have more than one profile for your users do the same procedure"

Reply:
Updated.